README.md

```diff
@@ -138,8 +138,9 @@ The following settings are stored in `template_config.yml`.
                         'github'. To switch from Redmine to GitHub use the --migrate-github-issues
                         option.
 
+  ci_update_branches    Specify the branches that should receive regular CI updates.
   keep_ci_update_since_branch
-                        Keep CI updated for all starting from the specified branch.
+                        Keep CI updated for all starting from the specified branch. DEPRECATED
```

Comment:
Can we just remove it? The worst case is plugin writers will not have some branches receive updates automatically, and then they would be like "oh I need to switch".

Reply:
I'm trying to follow the principle of least surprise.